tensorboard/components_polymer3/tf_line_chart_data_loader/tf-line-chart-data-loader.ts

```diff
@@ -31,13 +31,21 @@ import {DataLoaderBehavior} from '../tf_dashboard_common/data-loader-behavior';
 import '../vz_line_chart2/vz-line-chart2';
 import {YScaleType, FillArea} from '../vz_line_chart2/line-chart';
 
+export interface TfLineChartDataLoader extends HTMLElement {
```

Comment:
(Abbreviating as “`TLCDL`” and “`_TLCDL`” throughout.) Note that while
`TLCDL` is a type (an interface), `_TLCDL` is not a type but rather a
type constructor: that is, for any type `M`, `_TLCDL<M>` is a type that
implements `TLCDL`.  Each type `_TLCDL<M>` happens to have a member
`setSeriesMetadata` whose type depends on `M`, but since the interface
of `TLCDL` does not declare any such member, the method has no bearing
on the interface. It’s just an implementation detail, like any of the
other private methods.

This suffices because all call sites of `setSeriesMetadata` already type
the receiver as `any` (naturally, since this interface is new). I wrote
it this way to avoid the clutter of the type parameter, but if you
prefer I could try to expose its intended-public API more fully?